#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raise NameError exception"""
    raise NameError(message)
